src/dmd/mars.d

```diff
@@ -2125,3 +2168,278 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
     return errors;
 }
 
+
+private __gshared bool includeImports = false;
+// array of module patterns used to include/exclude imported modules
```

Comment:
I have no idea what a "module pattern" is.

Reply:
A module pattern is part of a module name.   I tried to make this clear in the associated documentation https://dlang.org/dmd-windows.html#switch-i[